### ISSUE TITLE
error scroll offset

### ISSUE THREAD

**Issue by ghost** [guillaumepotier/Parsley.js#903]
hi, 
nice to have - we got a sticky-nav on a page and need the option to define an offset for scrolling on error. thx

**Comment by marcandre** [comment]
PR accepted, especially if it also resolves #938

**Comment by marcandre** [comment]
Closing in favor of #938

**Comment by marcandre** [comment]
Mmm, sorry for the noise, this isn't quite the same. Reopening.
Have you solved your issue? I imagine that it wouldn't be too hard to do by catching the `form:error` event and scrolling some more if need be?

**Comment by Flaburgan** [comment]
Joining here, it would be nice to be able to give an offset to Parsley so it knows where it should scroll exactly.

**Comment by marcandre** [comment]
Yeah, the need is reasonable. Probably simply a new setting (say `headerHeight`?)
PR accepted

**Comment by chequille** [comment]
Is there already a parameter like headerHeight?
I am facing this problem (first error is not shown because header is on the foreground) currently and do have no idea how to fix it.
Does have somebody a sloution for this?
BR
Jürgen
